docs/examples/image_quality_and_benchmarks/structural_similarity_index.jl

```diff
@@ -0,0 +1,78 @@
+# ---
+# cover: assets/ssim.png
+# title: Structural Similarity Index
+# ---
+
+# When comparing images, the **Mean Squared Error** (or MSE), though straightforward to calculate,
+# may not be a very good indicator of their *perceived* similarity.
+
+# The **Structural Similarity Index** (or SSIM) aims to address this shortcoming by taking texture
+# into account, and assigning a higher score to images that may *appear* similar.
+
+
+using Images, TestImages
```

Comment:
```suggestion
using ImageQualityIndexes, ImageCore, TestImages
```
We can use ImageQualityIndexes, importing whole `Images `is not necesary. It would also give an idea to user that where the code resides.

Reply:
Will then have to have lots of imports, for example `ImageDistances.jl` for `mse`, `ImageView.jl` for `mosaicview` etc. Makes the example unwieldy imo.

Reply:
In that case its okay. Btw `mosaicview` is reexported by `ImageCore` too.